### DISCUSSION TITLE
File Uploader - Resize Image in Multiple Sizes

### DISCUSSION THREAD

**Discussion by Calbo67** [filamentphp/filament#10726] [Help]
I'm using the Filament file uploader and would like to upload an image and for that image to be resized to multiple different sizes and uploaded to the different folders. For example i upload an image then it resizes it to 500px and puts in in a folder called large then resizes the image to 250px and puts it in a folder called small. Is this possible with the uploader or do i need to do some custom logic.

**Comment by leandrocfe** [comment]
Spatie Media Library supports multiple conversions

https://filamentphp.com/plugins/filament-spatie-media-library
https://spatie.be/docs/laravel-medialibrary/v11/converting-images/defining-conversions#content-using-multiple-conversions

**Reply by Calbo67** [reply]
Thanks @leandrocfe i came across spatie library but i don't want to use a media table i have my own product_images table etc but couldn't configure it to save into a different table if you know how to?
